std_msgs/String bot1_mal
std_msgs/String bot2_mal
std_msgs/String bot3_mal
